src/test/resources/smartHouseFeatures/Mod3Case5/organization.feature

```diff
@@ -0,0 +1,13 @@
+# Login feature review the behavior in thie feature
+Feature: SHRemoveOrganization
+
+  Scenario: Home page is displayed once set credential in login page
+    Given M3C5: 'SMARTHOUSE' page is loaded
+    And M3C5: open 'Login' page
+    And M3C5: set admin credentials on 'Login' page
+    And M3C5: accept credentials on 'Login' page
+    And M3C5: close 'Login' page
+    And M3C5: open 'Organization' page
+    And M3C5: open 'Organization' detail
+    And M3C5: remove child 'Organization'
+    And M3C5: 'SMARTHOUSE' page is closed
```

Comment:
According to the test case it should be:
Given I loging to 'SMARTHOUSE' page with properly credentials
When go to 'Estructura Organizacional' on 'Header' page
And navigate to 'Gerencia General' option on 'Dashboar/Left Panel/etc...' page
And click over 'Eliminar' button on 'Genencia General' page
Then 'Area Gerencia General tiene asociados Areas y Empleados activos' information message should be displayed